Examples/SwiftUI Example/Beacon SwiftUI Example/SettingsView.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import SwiftUI
+import Beacon
+
+struct SettingsView: View {
+
+    var body: some View {
+        return Button("Contact Us") {
+            #warning("You must provide a valid Beacon ID")
```

Comment:
⚠️ Let's update the ReadMe's "Sample Application" section to be plural (both the title of the sub-section and the folder name) and then include sections for both the Objective-C and Swift UI examples. We no longer need the bit about running "pod install" for Obj-C and we can include a note about adding the Beacon ID on the line below for Swift UI.

Reply:
Great call.  Should be good to go now @codenameDuchess.